recipes/gnuplot/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+./configure \
+    --prefix=$PREFIX \
+    --without-x \
+    --without-lua
```

Comment:
Complaining cairo isn't found, so we may want `--without-cairo` or to include cairo as a dep.